test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -0,0 +1,358 @@
+/*
+*
+*  Copyright (c) 2015 Intel Corporation. All rights reserved.
+*  This software is available to you under the BSD license. For
+*  license information, see the LICENSE file in the top level directory.
+*
+*/
+
+#include <common.h>
+
+#define MAX_MSG_SIZE (1<<23)
+#define START_LEN 1
+
+#define INC 2
+#define TRIALS 500
+#define WINDOW_SIZE 64
+#define WARMUP 50
+
+#define TRIALS_LARGE  100
+#define WINDOW_SIZE_LARGE 64
+#define WARMUP_LARGE  10
+#define LARGE_MESSAGE_SIZE  8192
+
+#define TRUE  (1)
+#define FALSE (0)
```

Comment:
agreed; I thought about this at one point, but I think the thought got lost somewhere along the line, thanks for pointing it out